### PR TITLE
add links to prev/next posts to bottom of each blog post page

### PR DESCRIPTION
**Pull request by cvan**
image

### CODE REVIEW

File: themes/aframe/layout/blog.ejs [hunk 1]
```diff
@@ -31,6 +31,18 @@
         <%- page.content %>
       </article>
 
+      <footer class="footer guide-links c">
+        <% // Yes, these are correct. %>
```
Comment (ngokevin) [inline themes/aframe/layout/blog.ejs]: I don't think we need this

**Comment by ngokevin** [comment]
r+ noice

File: themes/aframe/layout/blog.ejs (hunk 1 above)
Comment (cvan) [inline themes/aframe/layout/blog.ejs]: ok, well it's backwards so it's a bit weird

Comment (ngokevin) [inline themes/aframe/layout/blog.ejs]: Ah I see that now.

**Pull request merged**